lib/server.js

```diff
@@ -1052,6 +1055,10 @@ WalletService.prototype.createTx = function(opts, cb) {
           excludeUnconfirmedUtxos: !!opts.excludeUnconfirmedUtxos,
         });
 
+        if (!self.clientVersion || _.startsWith(self.clientVersion, 'bwc-0.0.')) {
+          txp.version = '1.0.1';
```

Comment:
not sure we can change that now, but I would prefer to use a single integer for version, It will make things
easier.